templates/guides/migration-guide-1.0.0.md

```diff
@@ -0,0 +1,79 @@
+---
+page_title: "Fastly Terraform 1.0.0 Migration Guide"
+subcategory: "Migration"
+---
+
+## Fastly Terraform 1.0.0 Migration Guide
+
+**Resources omit `_v1` suffix**:
+
+All resources now have a consistent naming convention which omit the previous v1 suffix.
+
+- `fastly_service_v1` -> `fastly_service_vcl` (renamed with `_vcl` suffix)
+- `fastly_service_acl_entries_v1` -> `fastly_service_acl_entries`
+- `fastly_service_dictionary_items_v1` -> `fastly_service_dictionary_items`
+- `fastly_service_dynamic_snippet_content_v1` -> `fastly_service_dynamic_snippet_content`
+- `fastly_user_v1` -> `fastly_user`
+
+When switching to the new 1.0.0 release, some existing projects might see an error such as:
+
+```
+Error: no schema available for module.fastly.<...> while reading state; this is a bug in Terraform and should be reported
+```
+
+This error is because the Terraform state file will have a different resource name to what's now exposed via the 1.0.0 release. To resolve this issue you'll need to first remove the old resource from the state file and then reimport the data using the new resource name.
```

Comment:
```suggestion
This error occurs because the Terraform state file contains a different resource name to what's now exposed via the updated provider. To resolve this issue, remove the old resource from the state file and reimport the data using the new resource name.
```